source/extensions/filters/network/thrift_proxy/config.h

```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+#include <cstdint>
```

Comment:
not needed anymore i think

Reply:
👍 yes, it is not needed. I just forget to remove it.